packages/timeline-state-resolver-types/src/vizMSE.ts

```diff
@@ -185,8 +185,9 @@ export interface TimelineObjVIZMSECleanupShows extends TSRTimelineObjBase {
 		deviceType: DeviceType.VIZMSE
 		type: TimelineContentTypeVizMSE.CLEANUP_SHOWS
 
-		/** IDs of the Shows to cleanup */
+		/** IDs of the Shows to cleanup - is ignored if 'cleanupAllShows' is true */
 		showIds: string[]
+		cleanupAllShows: boolean
```

Comment:
I think ignoring one parameter based on the value of another can be a bit confusing. How about doing it the same way as in `VizMSECommandClearAllEngines`?
```suggestion
		/** IDs of the Shows to cleanup. Will clean ALL the shows in the MSE if set to 'all' */
		showIds: string[] | 'all'
```

Reply:
Hmm, coming from Java I'm not used to assign attributes with multiple types (hence why I didn't think of it).
I will give it some though, but most likely implement your suggestion as I kinda agree with the confusing part of ignoring attributes based on other attributes. (And boolean flags should in general be avoided)

Reply:
Updated